docs/run-command.md

```diff
@@ -9,11 +9,18 @@ of running scripts in an Azure Linux VM also apply here.
 ## Regardless of the Kubernetes control plane status
 
 When executing `run-command` without passing through Kubernetes, we need to provide the information of the node (VMSS instance) where we want to run the given command. To retrieve such information, we can use the [`config import`](./config.md#importing-configuration) command. Once we got it, we can select the node we want to use and all the subsequent `run-command` commands will be executed on that node:
-
 ```bash
+# Clean the current configuration (if any)
```

Comment:
I think comments should now become normal text and it should always looks like:

	  Now, let's do ...
	  -bash commands-
	  
	  In a cluster with three nodes, the output will look like this:
	  <!!--expected_similarity=X-->
	  -output-
	  
	  Next, let's do that ...
	  -bash commands-
	  
	  And this time the output ...
	  <!!--expected_similarity=X-->
		  -output-

Instead of a set of consecutive code blocks as it is now:

	  -bash commands-
	  <!!--expected_similarity=X-->
	  -output-

	  -bash commands-
	  <!!--expected_similarity=X-->
	  -output-

	  -bash commands-
	  <!!--expected_similarity=X-->
	  -output-